inc/Engine/Preload/Links/Subscriber.php

```diff
@@ -181,6 +181,9 @@ private function get_uris_to_exclude( $use_trailing_slash ) {
 			$uris .= '|' . str_replace( $site_url, '', $uri );
 		}
 
+		// exclude URL patterns used by referral plugins.
+		$uris .= '|/refer/|/go/';
```

Comment:
what do u think about adding a filter here so we can change with compatibility files whenever needed?
I believe that some plugins uses `recommend` and others open it to the user so he can change it, for this reason, using a filter will help a lot.

Reply:
Sorry I didn't see @piotrbak comment above :D